src/Microsoft.Identity.Web.Certificateless/ManagedIdentityClientAssertion.cs

```diff
@@ -47,41 +48,5 @@ private async Task<ClientAssertion> GetSignedAssertionFromFederatedTokenProvider
                 cancellationToken).ConfigureAwait(false);
             return new ClientAssertion(result.Token, result.ExpiresOn);
         }
-
```

Comment:
Moved these to the base class (as they are also used for the PodIdentityClientAssertion)